### PR TITLE
sys-kernel/bootengine: Bump to fix slow PXE/ISO boots with udev rule

### PR DESCRIPTION
**Pull request by chewi**
# sys-kernel/bootengine: Bump to fix slow boots with udev rule

See: flatcar/bootengine#98
See: https://github.com/flatcar/Flatcar/issues/1514

## How to use

See: flatcar/bootengine#98

## Testing done

Lots of manual testing and a packages_all_arches Jenkins job, as mentioned in the other PR. I'll do a GitHub run for this PR.

- [X] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [X] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.

### CODE REVIEW

File: sdk_container/src/third_party/coreos-overlay/sys-kernel/bootengine/bootengine-9999.ebuild
Comment (jepio) [inline sdk_container/src/third_party/coreos-overlay/sys-kernel/bootengine/bootengine-9999.ebuild:10]: needs to be d2e22122e8e106a21f64c048992290edc7053384

**Review by jepio — CHANGES_REQUESTED** [review]
tip: I usually avoid using `Closes:` tags in commit messages, and prefer cross-referencing with `See:`. The issue has now been closed even though we haven't merged the scripts PR yet. Even after this PR is merged, we'd still need to backport to stable-3975 (because that's where the issue was spotted) before we can consider the issue closed. Sometimes it even makes sense to wait until a release has been done.

**Review by krnowak — APPROVED** [review]
If this needs backporting to other channels, make sure you don't backport additional changes in bootengine by accident. If other channels were using different commit for bootengine, we will need to create backport branches in baselayout that contain your fix only.

**Comment by chewi** [comment]
Oh, sorry about that. Still getting my head around this. I thought alpha just tracked main.

**Pull request merged**
